contracts/.openzeppelin/kovan.json

```diff
@@ -234,6 +245,13 @@
         "implementation": "0x02d4ff4F4aed83d85914cA8A8eC988Bf7A520536",
         "admin": "0xbC3d397D8689B4A6B8047f813e1d40171A1B61b0",
         "kind": "Upgradeable"
+      },
+      {
+        "address": "0xb4fA1f818fa5B29F51Beb6FFA3941e0C5aCd6Ecd",
+        "version": "2.5.3",
+        "implementation": "0xbAa43dA970987ad0499e325bD1dA230ef8AEe5CF",
+        "admin": "0xbC3d397D8689B4A6B8047f813e1d40171A1B61b0",
+        "kind": "Upgradeable"
```

Comment:
I'd prefer to only ever have one version of the contract in here, it is confusing having multiple @moose-code